data/brands/amenity/fuel.json

```diff
@@ -3638,6 +3638,16 @@
         "name": "OK"
       }
     },
+    {
+      "displayName": "OK",
```

Comment:
They would have to have unique display names, eg:
```suggestion
      "displayName": "OK (NL)",
```

Reply:
Thanks!